src/pages/ProductsPage/ProductCard/ProductCardComponent.tsx

```diff
@@ -0,0 +1,132 @@
+import Button from '@mui/material/Button';
```

Comment:
try to use short syntax
import {
  Box,
  Typography ,
  CardMedia ,
  CardContent ,
  CardActions ,
  Card ,
} from '@mui/material';

Reply:
Import model has been simplified using the curly braces